src/main/java/au/gov/nla/dlir/models/event/Correction.java

```diff
@@ -0,0 +1,15 @@
+package au.gov.nla.dlir.models.event;
+
+import lombok.Getter;
+import lombok.Setter;
+import lombok.ToString;
+
+@Getter
+@Setter
+@ToString
+public class Correction {
+  private String type;
+  private Integer lineNumber;
+  private String correctedText;
+  private Integer beforeLineNumber;
```

Comment:
This might need to be a `String`
![image](https://user-images.githubusercontent.com/13306484/137054555-1a649e3e-6459-44ac-98d6-3c9c5f222170.png)

Also need to think about how to represent "after last line"?

Reply:
Does newspapers have an id for each line, or just the position of the line within a paragraph?